### PR TITLE
MF36 - Add support from rabbitmq

### LINKED ISSUE #36: Add Support For Multi-broker Messaging From Mainflux
**FEATURE REQUEST**

1. Is there an open issue addressing this request? No

2. Describe the feature you are requesting, as well as the possible use case(s) for it.
   Since mainflux now supports multi-broker messaging, it would be nice to have a way to specify the broker to use when publishing messages. This would allow for a more flexible deployment of mainflux.

3. Indicate the importance of this feature to you (must-have, should-have, nice-to-have).
   Should-have

### PR DESCRIPTION
**Pull request by SammyOina**
Signed-off-by: SammyOina <sammyoina@gmail.com>

### What does this do?
Add support for multiple brokers

### Which issue(s) does this PR fix/relate to?
Resolves #36

### Did you document any new/modified functionality?
https://github.com/mainflux/docs/pull/152

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ type MQTT struct {
 }
 
 type Server struct {
-	NatsURL   string `json:"nats" toml:"nats" mapstructure:"nats"`
+	BrokerURL string `json:"broker" toml:"broker" mapstructure:"broker"`
```
Comment (rodneyosodo) [inline pkg/config/config.go]:
```suggestion
	BrokerURL string `json:"broker_url" toml:"broker_url" mapstructure:"broker_url"`
```

File: pkg/export/service.go [hunk 2]
```diff
@@ -47,12 +49,13 @@ const (
 	NatsAll     = ">"
```
Comment (rodneyosodo) [inline pkg/export/service.go]: This should change

**Review by rodneyosodo — CHANGES_REQUESTED** [review]

File: pkg/export/service.go (hunk 2 above)
Comment (SammyOina) [reply pkg/export/service.go]: messaging package replaces this with # for rabbitmq so its not an issue

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**